scripts/restart-server

```diff
@@ -137,10 +142,10 @@ if action == "start" or args.less_graceful:
     logging.info("Starting workers")
     subprocess.check_call(["supervisorctl", "start", *workers])
 
-using_sso = subprocess.check_output(["./scripts/get-django-setting", "USING_APACHE_SSO"])
-if using_sso.strip() == b"True":
-    logging.info("Restarting Apache WSGI process...")
-    subprocess.check_call(["pkill", "-f", "apache2", "-u", "zulip"])
+    using_sso = subprocess.check_output(["./scripts/get-django-setting", "USING_APACHE_SSO"])
+    if using_sso.strip() == b"True":
+        logging.info("Restarting Apache WSGI process...")
+        subprocess.check_call(["pkill", "-f", "apache2", "-u", "zulip"])
```

Comment:
This hunk doesn't look right -- this looks like it only checks for Apache `if action == "start" or args.less_graceful`.

I think this needs its own `if os.path.exists("/etc/supervisor/conf.d/zulip/zulip.conf")` or can be moved up to just after Django -- I don't think there are any strong constraints that point to where the right place to run it is.

Reply:
@alexmv done, thanks for the review! :)

Reply:
@alexmv The only problem with this setup might be: Is it guaranteed that apache is only needed when `/etc/supervisor/conf.d/zulip/zulip.conf` is present?

Reply:
At least in the Zulip world, yes.  (The only use we have of Apache is for the Apache REMOTE_USER authentication backend)

Reply:
@timabbott Ok, thanks :) So do you think that there is there anything left to do for this patch?